_includes/documentation/documentation-version-band.html

```diff
@@ -1,39 +1,56 @@
 <div class="full-width-bg blue-bg secondary-page-title-band">
   <div class="component-slim grid-wrapper">
     <div class="width-12-12">
-      <h3>Documentation for Version {{site.data.releases.operator[0].version}} - latest stable release</h3>
+      <h3>Strimzi {{site.data.releases.operator[0].version}} documentation (latest stable release)</h3>
     </div>
     <div class="width-6-12 width-12-12-m documentation-version-band">
       <p>
-        <strong class="text-caps">Strimzi Kafka Operators Documentation</strong>
+        <strong class="text-caps">Strimzi {{site.data.releases.operator[0].version}} documentation</strong>
+      </p>
+      <p>
+        <strong class="text-caps">Strimzi {{site.data.releases.operator[0].version}} Kafka operators</strong>
         <br/>
-        Overview guide: <a href="{{site.baseurl}}/docs/operators/latest/overview.html">{{site.data.releases.operator[0].version}} - latest stable release</a> <a href="{{site.baseurl}}/docs/operators/latest/full/overview.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a><br/>
-        Quick Start guide: <a href="{{site.baseurl}}/docs/operators/latest/quickstart.html">{{site.data.releases.operator[0].version}} - latest stable release</a> <a href="{{site.baseurl}}/docs/operators/latest/full/quickstart.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a><br/>
-        Deploying and Upgrading: <a href="{{site.baseurl}}/docs/operators/latest/deploying.html">{{site.data.releases.operator[0].version}} - latest stable release</a> <a href="{{site.baseurl}}/docs/operators/latest/full/deploying.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a><br/>
-        Configuring Strimzi: <a href="{{site.baseurl}}/docs/operators/latest/configuring.html">{{site.data.releases.operator[0].version}} - latest stable release</a> <a href="{{site.baseurl}}/docs/operators/latest/full/configuring.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a><br/>
-        Example custom resources: <a href="{{site.github_url}}/strimzi-kafka-operator/tree/{{site.data.releases.operator[0].version}}/examples">{{site.data.releases.operator[0].version}} - latest stable release</a>
-      </p><br/>
+        <a href="{{site.baseurl}}/docs/operators/latest/overview.html">Overview ({{site.data.releases.operator[0].version}})</a> <a href="/docs/operators/latest/full/overview.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> <br/>
+        <a href="{{site.baseurl}}/docs/operators/latest/quickstart.html">Quick Start guide ({{site.data.releases.operator[0].version}})</a> <a href="/docs/operators/latest/full/quickstart.html" title="Full documentation (New Window)" target="_blank"><i class="fas fa-external-link-alt"></i></a> <br/>
```

Comment:
Any reason why this one has the `guide` and others don't?

Reply:
I'll remove that. Don't think we need it